test.js

```diff
@@ -49,6 +49,19 @@ test('throw error if task throws', t => {
 	t.throws(list.run(), 'foo bar');
 });
 
+test('swallows error if task skipped', t => {
```

Comment:
Anything we can do to test this more thoroughly? This test title doesn't seem to make sense. I don't see an error being thrown.